spec/classes/ops_manager_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe 'mongodb::opsmanager' do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) { facts }
+
+      let :pre_condition do
```

Comment:
This can be replaced with:
```ruby
let(:params) do
  {
    opsmanager_url: 'http://localhost:8080'
  }
end
```

Reply:
Just this, and a squash/rebase, and we're good to merge I think?